receiver/mongodbatlasreceiver/README.md

```diff
@@ -1,33 +1,50 @@
 # MongoDB Atlas Receiver
 
 Receives metrics from [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) 
-via their [monitoring APIs](https://docs.atlas.mongodb.com/reference/api/monitoring-and-logs/)
+via their [monitoring APIs](https://docs.atlas.mongodb.com/reference/api/monitoring-and-logs/),
+as well as alerts via a configured [webhook](https://www.mongodb.com/docs/atlas/tutorial/third-party-service-integrations/).
 
-Supported pipeline types: metrics
+Supported pipeline types: metrics, logs
```

Comment:
There's a new component readme header format that you'll need to use instead. See #10006